externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/ui/EngineSessionPage.scala

```diff
@@ -40,6 +45,34 @@ case class EngineSessionPage(parent: EngineTab)
       val sessionStat = store.getSession(parameterId).getOrElse(null)
       require(sessionStat != null, "Invalid sessionID[" + parameterId + "]")
 
+      val redactionPattern = parent.sparkUI match {
+        case Some(ui) => Some(ui.conf.get(SECRET_REDACTION_PATTERN))
+        case None => SECRET_REDACTION_PATTERN.defaultValue
+      }
+
+      val sessionPropertiesTable =
+        if (sessionStat.conf != null && !sessionStat.conf.isEmpty) {
+          val table = UIUtils.listingTable(
+            propertyHeader,
+            propertyRow,
+            Utils.redact(redactionPattern, sessionStat.conf.toArray),
```

Comment:
nit: sort the conf to make the ordering more readable

Reply:
done